### PR TITLE
Add api collection and some context to the repo

### PR DESCRIPTION
**Pull request by shantilal-hopin**
[INTP-1052](https://hopin.atlassian.net/browse/INTP-1052)

Added Postman collection of External API

Added README.md

Screenshot 2022-04-15 at 6 02 03 PM

### CODE REVIEW

File: Hopin API Collection.postman_collection.json [hunk 1]
```diff
@@ -0,0 +1,1153 @@
...
+                "tickets",
+                "{{ticketId}}",
+                "magicLinks"
+              ]
+            }
+          },
+          "response": []
+        },
+        {
+          "name": "Deletes Magic Link for given ticket type",
```
Comment (pulpecja) [inline Hopin API Collection.postman_collection.json]: we don't send ticket type here I think?

File: Hopin API Collection.postman_collection.json [hunk 2]
```diff
@@ -0,0 +1,1153 @@
...
+          },
+          "response": []
+        }
+      ]
+    },
+    {
+      "name": "Data Subscription",
+      "item": [
+        {
+          "name": "Create a new data subscription",
```
Comment (pulpecja) [inline Hopin API Collection.postman_collection.json]: `for user's organization's event`? or maybe `for an event`? we're not really consistent here

File: Hopin API Collection.postman_collection.json [hunk 3]
```diff
@@ -0,0 +1,1153 @@
...
+          },
+          "response": []
+        }
+      ]
+    },
+    {
+      "name": "Reports",
+      "item": [
+        {
+          "name": "Get Reports List",
```
Comment (pulpecja) [inline Hopin API Collection.postman_collection.json]: `for an event`

File: Hopin API Collection.postman_collection.json [hunk 4]
```diff
@@ -0,0 +1,1153 @@
...
+                "organizations",
+                "{{organizationId}}"
+              ]
+            },
+            "description": "Once you have the external ID for the Organization you will need to update the variable value for {organizationId} under the collection folder titled \"Hopin API Collections\".\n\nClick \"Hopin API Collections\" \"Variables\" Update the {organizationId} \"Current Value\" to your Org ID Click \"Save\""
+          },
+          "response": []
+        }
+      ],
+      "description": "Use these endpoint to retrieve the list of organizations property of the user"
```
Comment (pulpecja) [inline Hopin API Collection.postman_collection.json]: `this` :)

**Comment by mihail-stanchev-hopin** [comment]
Where do you think we can describe all the variables used in here like `{LocUrl}`, `{ProdURL}`, `{eventId}` and so on? It would be hard to begin using this collection without setting these up first.

**Comment by pulpecja** [comment]
> Where do you think we can describe all the variables used in here like `{LocUrl}`, `{ProdURL}`, `{eventId}` and so on? It would be hard to begin using this collection without setting these up first.

I'd say that ProdURL may be even set from the beginning?

**Review by pulpecja — APPROVED** [review]

**Comment by shantilal-hopin** [comment]
> 

Yeah, we have set its value by default, although I'll change the Var name(with some prefix) and describe the way to change its value in the collection description.

File: README.md [hunk 5]
```diff
@@ -1 +1,13 @@
...
+Repository containing the Postman collection of the Hopin External APIs, for more info about APIs, visit https://developer.hopin.com/external-api
+
+# Import Collection
+
+
+You can download `Hopin API Collection.postman_collection.json` file and import this file inside the Postman app as a collection.
+You may need to set some variables to execute the APIs as description given below.
+
+Once you have the Organization External ID you will need to update the variable value for organizationId under the collection folder titled `Hopin API Collection`.
+Click `Hopin API Collection` > `Variables` and Update the `organizationId` with your Organization external ID and Click `Save`, the same you you can change/override any variable value.
```
Comment (pulpecja) [inline README.md]: I would change this line to this:
```
Click `Hopin API Collection` > `Variables`, update the `organizationId` with your Organization external ID and click `Save`. You can do the same to change/override any variable value.
```

File: Hopin API Collection.postman_collection.json [hunk 6]
```diff
@@ -0,0 +1,1163 @@
+{
+  "info": {
+    "_postman_id": "b30bce21-92e6-4a66-823f-0be4ff103ac3",
+    "name": "Hopin API Collection",
+    "description": "This is the collection of the Hopin External APIs, for more info about APIs, visit [https://developer.hopin.com/external-api](https://developer.hopin.com/external-apiYou)\n\nYou may need to set some variables to execute the APIs as description given below.\n\nOnce you have the ID for the Organization you will need to update the variable value for `organizationId` under the collection folder titled `Hopin API Collection`.  \nClick `Hopin API Collection` > `Variables` Update the `organizationId` to your Organization external ID Click `Save`, the same you you can change/override any variable value.",
```
Comment (pulpecja) [inline Hopin API Collection.postman_collection.json]: I think similar change should be made here (look at my other comment :))

**Pull request merged**